docs/telemetry/adapters/gnmireverse/index.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: default
+title: "gNMIReverse Examples"
+date: 2021-05-19 20:03:00 --0600
+categories:
+---
+
+- [Introduction](#introduction)
+- [Prerequisite](#prerequisite)
+- [Installing and configuring gNMIReverse client](#installing-and-configuring-gnmireverse-client)
+    - [Building the client and server binaries](#building-the-client-and-server-binaries)
+- [gNMIReverse client daemon Configuration](#gnmireverse-client-daemon-configuration)
+    - [Non-default VRF](#non-default-vrf)
+    - [Default VRF](#default-vrf)
+- [Running the gNMIReverse Server](#running-the-gnmireverse-server)
+- [Subscribing to eos_native paths](#subscribing-to-eos_native-paths)
+- [Subscribing to Smash paths](#subscribing-to-smash-paths)
+- [References](#references)
```

Comment:
this can be removed.  the TOC will be automatically generated by mkdocs.

Reply:
Removed TOC from `docs/telemetry/adapters/gnmireverse/index.md`